libs/cloudsdk/README.md

```diff
@@ -1 +1,100 @@
-## Cloud SDK Library
+# Cloud SDK Library
+
+###cloudsdk.py : This Library Consists of the following-
+    1. class ConfigureCloudSDK : Base Configuration Class
+    2. class CloudSDK(ConfigureCloudSDK) : Main Cloudsdk Class
+    3. class ProfileUtility : Used to CRUD over CloudSDK Profiles Utility
+    4. class JFrogUtility : Used for Artifactory Utils, Get latest Build, Upload Firmware etc.
+
+###Note: cloudsdk.py has libraries that uses Swagger Autogenerated Code. 
+### Setup The Environment For using the cloudsdk library
+
+Using Swagger Autogenerated CloudSDK Library pypi package (implemented with [swagger codegen](https://github.com/swagger-api/swagger-codegen)).
+Using [pytest] as the test execution framework.
+Using [pylint](http://pylint.pycqa.org) for code quality monitoring.
+Using [allure](https://docs.qameta.io/allure/#_about) with Github Pages to report test outcome.
+
+### Follow the setps below to setup the environment for your development Environment
```

Comment:
Shouldn't this information in the top level Readme.md file?

Reply:
sure, we will add there